pkgs/os-specific/linux/kernel/hardened/update.py

```diff
@@ -158,7 +158,7 @@ def find_asset(filename: str) -> str:
 
 def parse_version(version_str: str) -> Version:
     version: Version = []
-    for component in re.split('\.|\-', version_str):
+    for component in re.split(r'\.|\-', version_str):
```

Comment:
Is this to do with the Python upgrade?

Reply:
It was never correct, but older Pythons quietly accepted it, and 3.12 at least warns.